common/it/com/commercetools/sunrise/common/version/UserAgentIntegrationTest.java

```diff
@@ -0,0 +1,61 @@
+package com.commercetools.sunrise.common.version;
+
+import com.commercetools.sunrise.it.WithSphereClient;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereClientConfig;
+import io.sphere.sdk.client.SphereClientFactory;
+import io.sphere.sdk.http.HttpClient;
+import io.sphere.sdk.http.HttpHeaders;
+import io.sphere.sdk.http.HttpRequest;
+import io.sphere.sdk.http.HttpResponse;
+import io.sphere.sdk.meta.BuildInfo;
+import io.sphere.sdk.projects.queries.ProjectGet;
+import org.junit.Test;
+
+import javax.annotation.Nullable;
+import java.util.concurrent.CompletionStage;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class UserAgentIntegrationTest extends WithSphereClient{
+
+
+    @Test
+    public void userAgent() throws Exception {
+        final SphereClientConfig config = sphereClientConfig();
+        try (final FakeHttpClient httpClient = new FakeHttpClient()) {
+            try (final SphereClient client = SphereClientFactory.of(() -> httpClient).createClient(config)) {
+                client.execute(ProjectGet.of()).toCompletableFuture().join();
+                assertThat(httpClient.getLastUserAgent())
+                        .contains("sunrise-java-shop-framework");
```

Comment:
It's a pity we cannot check whether the version is there. Maybe also that the JVM SDK part is included too.

Having a user-agent that looks like this:
```
commercetools-jvm-sdk/1.13.0 (AHC/2.0) Java/1.8.0_92-b14 (Mac OS X; x86_64) sunrise-java-shop-framework/1.0.0-M4-SNAPSHOT
```
We could maybe check `commercetools-jvm-sdk` is there and `sunrise-java-shop-framework` as we have it, and with the thing I explain in the next comment we could even check the version. WDYT?